test/test.hs

```diff
@@ -3953,8 +3976,7 @@ tests = testGroup "hevm"
                           _ -> False
           assertBoolM "Did not find expected storage cex" testCex
           putStrLnM "expected counterexample found"
-        ,
-        expectFail $ test "calling unique contracts (read from storage)" $ do
+        , test "calling-unique-contracts--read-from-storage" $ do
```

Comment:
Can you explain what changed here?
What exactly is the this test checking?
What was happening before and what is happening now?